### ISSUE TITLE
[FEATURE] MLflow tests

### ISSUE THREAD

**Issue by rodrigo-arenas** [rodrigo-arenas/Sklearn-genetic-opt#30]
**Is your feature request related to a problem? Please describe.**
Currently there are not unit tests to the integration with MLflow

**Describe the solution you'd expect**
Create the file in sklearn_genetic/tests/test_mlflow.py and put the set of test that contains the use case of MLflow from sklearn_genetic.mlflow
It should test if the config creates or no a new topic and the use of each parameter, as well, that at the end of the runs the logged artifacts/metric/hyperparameters exists in the mlflow server and clean the resources after the test is ended

**Comment by Turtle24** [comment]
Hey, I'd like to take this one on. We've been working with MLflow recently so I'd like to get more into it.

**Comment by rodrigo-arenas** [comment]
Hey, that is great! Let me know if you have any doubts about the package itself

**Comment by rodrigo-arenas** [comment]
Thanks to @Turtle24 now there are tests on mlflow, we'are still looking how to run them using docker as the mlflow server to make sure we don't mess up some local mlflow instance
